### ISSUE TITLE
CSGhost v4.2.1 needs an update I guess

### ISSUE THREAD

**Issue by johnny1312x** [KittenPopo/csghost-faq#4]
CSGhost doesn't work after new Steam update.
I'm using it since v2 and I wish I can use it for a very long time. 
In fact, it is the only VB and injector I trust.

So please Kitten, solve it and maybe add some paid subs to it, maybe we can pay you work doing this shit really good.

**Comment by aki6996** [comment]
yeah i would like to know if Kitten can fix this at all and how long it would take

**Comment by johnny1312x** [comment]
> 

he was able to create it, there's no doubt about fixing it...but no one knows when.

**Comment by lolz74** [comment]
Well v4.3 just came so i guess it doesnt need an update ¯\_(ツ)_/¯

**Comment by Arctrometa** [comment]
> Well v4.3 just came out so i guess it doesnt need an update ¯_(ツ)_/¯

hey can you give me the link for v4.3, I'm not able to find it anywhere?

**Comment by johnny1312x** [comment]
> 

can you leave a link or something here? we can't find nothing about v4.3.

**Comment by lolz74** [comment]
go on UC, although you have to wait a bit since it hasn't been completely approved by moderators yet.

Link: https://www.unknowncheats.me/forum/cs-go-releases/454734-csghost-v4-injector-integrated-vac-bypass.html

**Comment by johnny1312x** [comment]
> go on UC, although you have to wait a bit since it hasn't been completely approved by moderators yet.
> 
> Link: https://www.unknowncheats.me/forum/cs-go-releases/454734-csghost-v4-injector-integrated-vac-bypass.html

well thank you very much! Have a great day!
